### ISSUE TITLE
Client tool error

### ISSUE THREAD

**Issue by fallanic** [maxogden/simplify-geojson#7]
Hi,

I just wanted to try your tool, it looks like the client tool doesn't work when the module is installed with `npm install -g`

```
fabiens-MBP-2:Downloads chef$ cat countries.geo.json | simplify-geojson -t 0.01

fs.js:439
  return binding.open(pathModule._makeLong(path), stringToFlags(flags), mode);
                 ^
Error: ENOENT, no such file or directory './usage.txt'
    at Object.fs.openSync (fs.js:439:18)
    at Object.fs.readFileSync (fs.js:290:15)
    at Object.<anonymous> (/Users/chef/.nvm/v0.10.41/lib/node_modules/simplify-geojson/cli.js:7:16)
    at Module._compile (module.js:456:26)
    at Object.Module._extensions..js (module.js:474:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:312:12)
    at Function.Module.runMain (module.js:497:10)
    at startup (node.js:119:16)
    at node.js:935:3
```

https://github.com/maxogden/simplify-geojson/blob/db165e0814455de5eaf3ef8e6e5a48f096face28/cli.js#L7

Thanks.

**Comment by maxogden** [comment]
thanks, i just fixed it and released 1.0.2

**Comment by fallanic** [comment]
Awesome thank you.
